src/main/java/gov/cms/madie/services/MeasureMapper.java

```diff
@@ -128,7 +128,7 @@ public interface MeasureMapper {
   @Mapping(
       target = "measurePopulationExclusionsDescription",
       expression =
-          "java(gov.cms.madie.util.MappingUtil.getPopulationDescription(measure, PopulationType.MEASURE_POPULATION_OBSERVATION))")
+          "java(gov.cms.madie.util.MappingUtil.getPopulationDescription(measure, PopulationType.MEASURE_POPULATION_EXCLUSION))")
```

Comment:
Question: Why the change to the constant?

Reply:
It was looking at the wrong thing! It was noticed by nate